stable/admin/README.md

```diff
@@ -200,9 +200,11 @@ The following tables list the configurable parameters for the `admin` option of
 | `tlsKeyStore.secret` | TLS keystore secret name | `nil` |
 | `tlsKeyStore.key` | TLS keystore secret key | `nil` |
 | `tlsKeyStore.password` | TLS keystore secret password | `nil` |
+| `tlsKeyStore.passwordKey` | TLS keystore secret password key. One of `tlsKeyStore.password` or `tlsKeyStore.passwordKey` must be used | `nil` |
```

Comment:
This is misleading, because 0 can be used. I would change the wording to "At most one..."

Reply:
Why don't we just set a default of `password` for the password key and only use it if TLS is enabled and `tlsKeyStore.password` is `nil`?

Reply:
So it looks like actually the default value of `tlsKeyStore` is `nil`, and any sub-keys within it are what indicate that TLS is enabled. I guess we can inject the `password` in the place where the environment variable is defined.

Reply:
If TLS is disabled, then `admin.tlsKeyStore`, `admin.tlsTrustStore`, `admin.tlsClientPEM`, and `admin.tlsCACert` are `nil`.  This change adds another optional property `passwordKey` to `admin.tlsKeyStore` and `admin.tlsTrustStore` Helm variables so that the user can supply passwords via Secret. Are you proposing to set a default value for this new property (i.e. `password`) in [here](https://github.com/nuodb/nuodb-helm-charts/blob/sivanov/HELM-389/stable/admin/templates/statefulset.yaml#L124-L132)?

Reply:
Yes, basically replacing [line 131](https://github.com/nuodb/nuodb-helm-charts/blob/sivanov/HELM-389/stable/admin/templates/statefulset.yaml#L131) with
```
              key: {{ default "password" .Values.admin.tlsKeyStore.passwordKey }}
```

But you would also have to propagate that to the places where the actual password is obtained like the `admin.keystorePassword`. But if that's too much of a pain, we can skip it.